File: deskover-dashboard-website/src/app/dtos/product-dto.ts
export interface ProductDto {
  id: number;
  name: string;
  slug: string;
  description: string;
  price: number;
  image: string;
  createdAt: Date;
  modifiedAt: Date;
  modifiedBy: string;
  actived: boolean;
  subcategoryId: number;
  brandId: number;
  discountId: number;
}

File: deskover-ecommerce-website/src/main/java/com/deskover/model/entity/dto/ProductDto.java
package com.deskover.model.entity.dto;

import java.io.Serializable;
import java.sql.Timestamp;

import com.deskover.model.entity.database.Product;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ProductDto implements Serializable{

	private static final long serialVersionUID = 8108059535055901960L;
    private Long id;
    private String name;
    private String slug;
    private String description;
    private Double price;
    private String image;
    private Long quantity;
    private Timestamp modifiedAt;
    private String modifiedBy;
    private Boolean actived;


}
